packages/synapse-react-client/src/components/TableColumnSchemaEditor/Validators/DatetimeSchema.ts

```diff
@@ -0,0 +1,21 @@
+import { z } from 'zod'
+import dayjs from 'dayjs'
+import utc from 'dayjs/plugin/utc'
+
+dayjs.extend(utc)
+
+const dateTimeString = z.string().transform(data => {
+  // Note: can't use parseInt on a timestamp! "2000-01-01T00:00:00Z" will be parsed as 2000
+  const isInt = data.match(/^\d+$/)
+  if (isInt) {
+    return dayjs.utc(parseInt(data))
```

Comment:
not calling toISOString() in this case?

Reply:
This was an oversight, so I will change it. It ultimately gets re-wrapped in `dayjs` and `toIsoString` is called again, so it doesn't change the behavior of the exported validator.